.pre-commit-config.yaml

```diff
@@ -46,3 +46,8 @@ repos:
       pass_filenames: false
       entry: "dade-copyright-headers update"
       types: [text]
+    - id: platform-independence-dar-check
```

Comment:
I worry that non-compiler devs will find this check annoying but I couldn't find a way to skip it by default that allows un-skipping it

Reply:
env vars! thank you @garyverhaegen-da, I've made the suggested changes